### PR TITLE
feat: add Uzbek(latn) locale

### PR DESCRIPTION
**Pull request by yoyo837**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [x] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

Same as #47797 wants to do.
close #47797
### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    add Uzbek(latn) locale      |
| 🇨🇳 Chinese |     添加乌兹别克语(拉丁字母) 国际化      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by yoyo837** [comment]
@Zukhrik

File: docs/react/i18n.zh-CN.md [hunk 1]
```diff
@@ -93,6 +93,7 @@ return (
 | 土库曼               | tk_TK  |
 | 乌尔都语 (巴基斯坦)  | ur_PK  |
 | 乌克兰语             | uk_UA  |
+| 乌兹别克语(拉丁字母) | uz_UZ  |
```
Comment (afc163) [inline docs/react/i18n.zh-CN.md:96]:
```suggestion
| 乌兹别克语（拉丁字母） | uz_UZ  |
```

Comment (yoyo837) [reply docs/react/i18n.zh-CN.md]: 给加点 size-limit 没过master不行

**Review by Zukhrik — APPROVED** [review]

**Comment by li-jia-nan** [comment]
@SocketSecurity ignore npm/fsevents@1.2.13

**Review by li-jia-nan — APPROVED** [review]

**Pull request merged**